cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/busbar_change/BusBarChangeProcessor.java

```diff
@@ -134,20 +134,20 @@ private static void createSwitches(Map<String, Set<SwitchPairToCreate>> switches
         // Get a list of all switch paris to create, some of them may be in common for different RAs
         List<SwitchPairToCreate> uniqueSwitchPairsToCreate = switchesToCreatePerRa.values().stream().flatMap(Set::stream).sorted().collect(Collectors.toList());
         // Store information about created fictitious buses for moving branches
-        Map<String, String> fictitiousBusIdPerBranchId = new HashMap<>();
+        Map<String, String> fictitiousBusIdPerBranchAndSide = new HashMap<>();
         for (SwitchPairToCreate switchPairToCreate: uniqueSwitchPairsToCreate) {
             String fictitiousBusId;
             // First, see if the branch has already been moved to a fictitious bus by a previous switch pair creation
-            if (fictitiousBusIdPerBranchId.containsKey(switchPairToCreate.branchId)) {
+            if (fictitiousBusIdPerBranchAndSide.containsKey(switchPairToCreate.branchAndSide())) {
                 // If yes, re-use the same fictitious bus
-                fictitiousBusId = fictitiousBusIdPerBranchId.get(switchPairToCreate.branchId);
+                fictitiousBusId = fictitiousBusIdPerBranchAndSide.get(switchPairToCreate.branchAndSide());
             } else {
                 // If not, create a new fictitious bus and store the info
                 fictitiousBusId = NetworkHelper.moveBranchToNewFictitiousBus(switchPairToCreate, networkModifier);
```

Comment:
What do you think about refactoring this method like that `networkModifier.moveBranchToNewFictitiousBus(switchPairToCreate.getBranchId(),switchPairToCreate.getBranchSideToModify());` ? To make it clearer